src/core/router/Routes.js

```diff
@@ -4,6 +4,14 @@
 
 const ROOT :'/' = '/';
 
+const ACCESS_ID_PARAM :string = ':accessId';
```

Comment:
I prefer `REQUEST_ID_PARAM = ':requestId';`. Also, please set the type to the exact string.